crmsh/bootstrap.py

```diff
@@ -509,6 +519,45 @@ def _get_sbd_device_from_config(self):
         else:
             return None
 
+    def _restart_cluster_on_needed(self):
```

Comment:
This method, at least with the current name is confusing. 
The method restarts the cluster but configures `sbd` too, which looks the main purpose. I think we need to find a new name.

Reply:
Changed as `_restart_cluster_and_configure_sbd_ra`